config-service/build.gradle.kts

```diff
@@ -10,8 +10,34 @@ plugins {
 
 dependencies {
   implementation(project(":config-service-impl"))
+
+  constraints {
+    implementation("com.google.guava:guava:30.0-jre") {
+      because("https://snyk.io/vuln/SNYK-JAVA-COMGOOGLEGUAVA-1015415")
+    }
+    runtimeOnly("io.netty:netty-codec-http2:4.1.53.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+    }
+    runtimeOnly("io.netty:netty-handler-proxy:4.1.53.Final") {
+      because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1020439")
+    }
+  }
+
+  implementation("org.hypertrace.core.grpcutils:grpc-server-utils:0.2.0")
+  implementation("org.hypertrace.core.serviceframework:platform-service-framework:0.1.13")
+
+  runtimeOnly("io.grpc:grpc-netty:1.30.2")
+  implementation("com.typesafe:config:1.4.0")
+
+  implementation("org.slf4j:slf4j-api:1.7.30")
+  runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:2.13.3")
 }
 
 application {
-  mainClassName = "org.hypertrace.config.service.MyApplication"
+  mainClassName = "org.hypertrace.core.serviceframework.PlatformServiceLauncher"
 }
+
+// Config for gw run to be able to run this locally. Just execute gw run here on Intellij or on the console.
+tasks.run<JavaExec>  {
+  jvmArgs = listOf("-Dbootstrap.config.uri=file:${projectDir}/src/main/resources/configs", "-Dservice.name=${project.name}")
```

Comment:
bootstrap config URI not needed, this will be picked up by default (I believe service.name is still needed though)

Reply:
Removed it